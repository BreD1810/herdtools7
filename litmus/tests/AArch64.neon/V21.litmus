AArch64 V21
(* Tests MOV scalar *)
{
0:X0=x;0:X1=y;0:X2=z;
}

 P0                     ;
 MOVI V0.8B,#1          ;
 MOV B1,V0.8B[0]        ;
 MOV B2,V0.8B[1]        ;
 MOV B3,V0.8B[8]        ;
 ST1 {V1.B}[0],[X0]    ;
 ST1 {V2.B}[0],[X1]    ;
 ST1 {V3.B}[0],[X2]    ;
 forall (x=1 /\ y=1 /\ z=0)
