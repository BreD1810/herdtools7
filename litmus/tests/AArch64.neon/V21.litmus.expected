Test V21 Required
Histogram (1 states)
1000000:>x=1; y=1; z=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=1 /\ y=1 /\ z=0) is validated
Hash=6dc68892ebeb701390f9f95b15289dd6
Observation V21 Always 1000000 0
